components/gitpod-protocol/src/util/logging.ts

```diff
@@ -308,7 +308,6 @@ function makeLogItem(severity: GoogleLogSeverity, context: LogContext | undefine
         severity,
         time: new Date().toISOString(),
         environment: process.env.KUBE_STAGE,
-        region: process.env.GITPOD_REGION,
```

Comment:
For future reviewers: we don't need to log the region out because that's added by the log collection infrastructure (e.g. Stackdriver)